tests/integration/utils2.go

```diff
@@ -656,6 +657,9 @@ func setStartingNodes(
 
 		s.nodes = append(s.nodes, c)
 		s.dbPaths = append(s.dbPaths, path)
+
+		sub := c.Events().Subscribe(100, events.MergeCompleteEventName)
```

Comment:
todo: Please document why `100`

Reply:
this is no longer relevant to the implementation